openlane/steps/netgen.py

```diff
@@ -100,17 +101,29 @@ class NetgenStep(TclStep):
     outputs = []
 
     config_vars = [
+        Variable(
+            "MAGIC_EXT_USE_GDS",
+            bool,
+            "A flag to choose whether to use GDS for spice extraction or not. If not, then the extraction will be done using the DEF/LEF, which is faster.",
+            default=False,
+        ),
         Variable(
             "NETGEN_SETUP",
             Path,
             "A path to the setup file for Netgen used to configure LVS. If set to None, this PDK will not support Netgen-based steps.",
             deprecated_names=["NETGEN_SETUP_FILE"],
             pdk=True,
         ),
+        Variable(
+            "NETGEN_INCLUDE_MARCOS_NETLIST",
```

Comment:
Will adjust the name and description.

> Why not just `NETGEN_LVS_HIERARCHICAL`…?

Because this variable alone doesn't lead to hierarchical LVS. You will need to enable `MAGIC_EXT_USE_GDS`.